app/src/main/java/app/revanced/manager/ui/component/bundle/ImportBundleTypeSelectorDialog.kt

```diff
@@ -0,0 +1,95 @@
+package app.revanced.manager.ui.component.bundle
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Divider
+import androidx.compose.material3.ListItem
+import androidx.compose.material3.RadioButton
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.unit.dp
+import app.revanced.manager.R
+import app.revanced.manager.ui.component.AlertDialogExtended
+import app.revanced.manager.ui.model.BundleType
+
+@Composable
+fun ImportBundleTypeSelectorDialog(
+    onDismiss: () -> Unit,
+    onConfirm: (BundleType) -> Unit,
+) {
+    var bundleType: BundleType by rememberSaveable { mutableStateOf(BundleType.Remote) }
+
+    AlertDialogExtended(
+        onDismissRequest = onDismiss,
+        confirmButton = {
+            TextButton(
+                onClick = { bundleType.let(onConfirm) }
```

Comment:
`bundleType` is not nullable and there is no long function chain, so `.let` isn't very useful here